core/sql/executor/HBaseClient_JNI.cpp

```diff
@@ -5619,3 +5394,41 @@ int convertStringObjectArrayToList(NAHeap *heap, jarray j_objArray,
     return arrayLen;
 }
 
+
+jint convertByteArrayObjectArrayToNAArray(NAHeap *heap, jarray j_objArray, NAArray<HbaseStr> **retArray)
+{
```

Comment:
For safety, might be a good idea to set retArray = 0 first thing in the method. Otherwise a caller might ignore an error and use the uninitialized pointer, with unpredictable results.